### PR TITLE
create artisan command

### PR DESCRIPTION
**Pull request by imanghafoori1**

### CODE REVIEW

File: src/Laravel/WebHookArtisanCommand.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        parent::__construct();
+    }
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'telegram:webhook
+                {--setup : To declare your webhook on telegram servers. So the can call you.}
```
Comment (jonnywilliamson) [inline src/Laravel/WebHookArtisanCommand.php]: Typo 

So THEY can call you

**Comment by jonnywilliamson** [comment]
Haven't checked this but it looks good. Nice job.

**Comment by jonnywilliamson** [comment]
Also, looks like one of the tests is failing.

It's being passed an object instead of an array. Probably a simple fix. But would be good to have it fixed if you can. Thanks

**Comment by jonnywilliamson** [comment]
@irazasyed is going to fall over when he sees the number of commits. It'll twitch at his OCD 😆 😀

**Comment by imanghafoori1** [comment]
Previous commits are already merged.

**Comment by imanghafoori1** [comment]
I do not know why is that failing...
I have created a new command class and added it in the service provider.
Is it introduced in the 2 recent commits?

**Comment by irazasyed** [comment]
Please create a new PR with only the new addition/changes. You might wanna rebase your repo and also create a new branch if required.

**Pull request closed without merging**